lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -63,6 +66,36 @@ def ajax_content_response(request, course_key, content):
     })
 
 
+def track_forum_event(request, event_name, course, obj, data):
+    """
+    Send out an analytics event when a forum event happens. Works for threads,
+    responses to threads, and comments on those responses.
+    """
+    user = request.user
+    data['id'] = obj.id
+    id_map = get_discussion_id_map(course)
```

Comment:
in `create_thread` it's internally during the call to `add_courseware_context`.  in `create_comment` i don't see it getting called.   if we must fetch the name here i think we should at least solve the double hit.  passing it to the relevant function(s) from outside seems like a reasonable way to go there.